### PR TITLE
fix: various ui improvements and bug fix

### PR DESCRIPTION
**Pull request by vadson71**
[Issue](https://github.wdf.sap.corp/ux-engineering/tools-suite/issues/31433)
CPE UI changes:
- improved warning texts on popups
- suppressed repetitive warning after preview reload
- improved quick actions styles (long action names are not truncated but wrapped to multiline representation)
- corrected distances between actions, and their groups
- fixed wrong positioning of the action context menu.

### CODE REVIEW

File: .changeset/great-glasses-pump.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/control-property-editor': patch
+---
```
Comment (Jimmy-Joseph19) [inline .changeset/great-glasses-pump.md:4]: need to add preview-middleware as well

Comment (vadson71) [reply .changeset/great-glasses-pump.md:5]: Thanks! Added

**Review by nikmace — CHANGES_REQUESTED** [review]

**Review by nikmace — APPROVED** [review]
Code looks good
Coverage is good
Did not test manually

**Review by Jimmy-Joseph19 — APPROVED** [review]
Thanks for the fixes. 
Changes looks good.
Test coverage good.
Haven't tested manually.
Approving.

**Pull request merged**